### DISCUSSION TITLE
0.10.8 (stable) /data files - Where can I find them?

### DISCUSSION THREAD

**Discussion by esksp** [endless-sky/endless-sky#10666] [Q&A] [answered]
(Sorry new to GitHub and not a programmer, so I'm not used to Git software...)

Anyway: My downloaded game (0.10.8) is apparently kinky (#10662), so I compared my /data files with the ones in this repository. Apparently there is a _huge_ difference: My own "map systems.txt" is a meager 730 kB, the one here is proud 1.8 MB. Same for the "map planets.txt" (487.3 kB vs. 1.5 MB). And I guess there are others too.

- So the obvious question is, what am I seeing? Is what I see here what will eventually appear in v.0.10.9? Or are those (still) the official 0.10.8 files?
- If not, where can I find the official, standard 0.10.8 files, so I can check what is wrong with my install and repair it?

**Comment by TheGiraffe3** [comment]
These are 0.10.9 files.
0.10.9 (which has been released) added two new alien species, and a whole bunch of systems.
So I wouldn't be surprised at the size.

**Comment by danaris** [comment]
Not sure where you're getting those sizes for the files here...?

The current `map systems.txt` is 738KB; `map planets.txt` is 550KB.

**Reply by TheGiraffe3** [reply]
These are the current `master` files, for the main continuous branch of the repository.
You and esksp have 0.10.8, which doesn't have as many systems.

**Reply by danaris** [reply]
No; I literally just pulled in the latest commits.

For `map planets.txt` to have bloated to 1.5MB, we would have needed to _triple_ the number of planets in the game.

**Reply by TheGiraffe3** [reply]
This is `map systems.txt`.

**Reply by danaris** [reply]
He listed sizes for both of them, as did I. I just picked one of them as an example.

**Reply by esksp** [reply]
I _does_ sound like a lot of new stuff for the file size to double, but then I don't know how busy you guys have been... :-D

Anyway, waiting for 0.10.9 to be released, is there any place I can find the correct, current files for my 0.10.8?

**Reply by TheGiraffe3** [reply]
It shouldn't be that much, you're right.
0.10.9 has been out for a few weeks.
https://github.com/endless-sky/endless-sky/releases/tag/v0.10.9

**Reply by danaris** [reply, marked as answer]
> I _does_ sound like a lot of new stuff for the file size to double, but then I don't know how busy you guys have been... :-D
> 
> Anyway, waiting for 0.10.9 to be released, is there any place I can find the correct, current files for my 0.10.8?

You can download a complete copy of 0.10.8 here: https://github.com/endless-sky/endless-sky/releases/tag/v0.10.8

**Reply by esksp** [reply]
> Not sure where you're getting those sizes for the files here...?

My bad. I tried to downloaded those files, but actually the things I downloaded, despite having the right names, are just JavaScript-stuffed HTML pages. So **no, the files are not 1.8 MB and 1.5 MB**.
Sorry. 
As I said, I don't know my way around GitHub...

**Reply by esksp** [reply]
> You can download a complete copy of 0.10.8 here: https://github.com/endless-sky/endless-sky/releases/tag/v0.10.8

Thanks!

**Comment by warp-core** [comment]
You can browse the repository as it was at the time of the 0.10.8 release here: https://github.com/endless-sky/endless-sky/tree/v0.10.8

In general, you can get to the state of the repository at any previous release by choosing the tag with the appropriate name from the list in the "tags" tab of hte branch drop-down menu on the main page for hte repository. The button is labelled "master" by default.

Like this: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/viewing-branches-in-your-repository
But using the "tags" tab in the drop down menu.

**Reply by esksp** [reply]
Thanks, but this is definitely above my pay grade... :-D
I'd be afraid to break something. I rather restrain myself to what I know, and ask for the rest--before going down in history as the guy who accidentally wiped the ES repository!... :-D

(Not so far-fetched, now I've found out I was accidentally basing my plugin on 0.10.6 files... I'm even wondering if I don't have to start my _actual_, vanilla game all over again too. After all my plugin overrode all the 0.10.8 star systems with 0.10.6 systems, and while I guess that didn't change the quests, it might have changed a lot of sight-seeing and environment stuff though, things I wouldn't like to miss...
Very annoying, I was through the Free Worlds, Pug, Hai, Wanderers story lines, had met the Coalition, had reached a 14k combat rating, it was August 3023... :-(
Oh well.)

**Reply by TheGiraffe3** [reply]
Saves are forwards and backwards-compatible, so you don't have to start a new one.

**Reply by esksp** [reply]
Yes, but I surely missed stuff...

**Reply by Hecter94** [reply]
> Saves are forwards and backwards-compatible, so you don't have to start a new one.

Forward compatibility, yes, backward compatibility isn't guaranteed and can often break.
